src/main/java/config/ConnectionConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package config;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionConfiguration {
+    private static final String URL = "jdbc:postgresql://localhost:5432/TicketServiceDB";;
+    private static final String USER_NAME ="postgres";
+    private static final String PASSWORD = "123";
```

Comment:
read these values from file, which you can put into resources folder

Reply:
Done